src/main/kotlin/no/nav/familie/ks/sak/kjerne/arbeidsfordeling/KontantstøtteEnhet.kt

```diff
@@ -30,5 +32,7 @@ enum class KontantstøtteEnhet(
             )
 
         fun erGyldigBehandlendeKontantstøtteEnhet(enhetsnummer: String): Boolean = GYLDIGE_BEHANDLENDE_KONTANTSTØTTE_ENHETER.any { it.enhetsnummer == enhetsnummer }
+
+        fun hentEnhet(enhetsnummer: String): KontantstøtteEnhet = entries.firstOrNull { it.enhetsnummer == enhetsnummer } ?: throw Feil("Finner ikke enhet med enhetsnummer $enhetsnummer")
```

Comment:
Småpirk, men hadde det vært bedre å kalle metoden `fraEnhetsnummer`? Tenkte dette først gjorde et nettverkskall siden det begynte med "hent" 😅 


```suggestion
        fun fraEnhetsnummer(enhetsnummer: String): KontantstøtteEnhet = entries.firstOrNull { it.enhetsnummer == enhetsnummer } ?: throw Feil("Finner ikke enhet med enhetsnummer $enhetsnummer")
```